### DISCUSSION TITLE
Why cpm of server differs from client's？

### DISCUSSION THREAD

**Discussion by 1393467261** [apache/skywalking#8072] [Q&A] [answered]
Server's cpm: 1150
![image](https://user-images.githubusercontent.com/32722524/140271418-11e5d216-455d-4ba8-a471-59f3d6db0743.png)
Client's cpm: 1891
![image](https://user-images.githubusercontent.com/32722524/140271374-697dd36e-2988-402a-84dd-1b9f73463c3f.png)

Since both cpms are collected from the same ServiceRelation, why cpm of client is significantly larger than server's？

**Comment by wu-sheng** [comment, marked as answer]
2 possibilities, maybe more
1. Your SkyWalking OAP servers are running in cluster mode, but not set correctly, then there is write overlap which causes data precision issue.
2. Client side faces lower successful rate, 404 or TLS(or others) make the server side doesn't process the requests, which will be ignored when CPM calculation.

**Reply by 1393467261** [reply]
Thanks, the first should be my case. And more discussion about the second possibility: The CPM calculation took place while parsing the entry span where both source service and dest service are avaliable, or also in the exit span parsing.

**Reply by wu-sheng** [reply]
CPM at client side is relating to exit span. In A->B case, A's exit span is the source of client side CPM.

**Reply by 1393467261** [reply]
Appreciate it.
